src/Utils/Utils.php

```diff
@@ -43,6 +42,16 @@ public static function removeControlCharacters(?string $string): string
         return $result ?? $string;
     }
 
+    /**
+     * Filter names are not allowed to have special characters inside of them. If they do, you may not be able
+     * to properly select a filter in certain scenarios. This is only relevant for internal naming, which is
+     * used for the query parameters. The naming in the customer-login can differ though.
```

Comment:
> A filter name should not contain special characters so it can be selected properly.

I would go with this sentence only, the other information doesn't help.

Reply:
That comment wouldn't make any sense if I would rename the method.

Reply:
Then please improve your current comment:

> certain scenarios

Which scenarios? This is not helpful.

> This is only relevant for internal naming, which is used for the query parameters. The naming in the customer-login can differ though

Does anybody care if naming in clogin is different?

Reply:
I removed the comment entirely in the last review, since it didn't make sense when the method name is `removeSpecialChars`. I am fine with that.